src/config.yml

```diff
@@ -123,6 +127,11 @@ blocked_commands:
   - 's:b:/lrbuild:_'
   - 's:b:/defaultgamemode:_'
   - 's:b:/reload:_'
+  # Soon to be blocked TFM Command i think!
+  - 's:b:/spectator:_' 
```

Comment:
This shouldnt be blocked. If its not already a custom TFM command then it should be created to create a spectator mode.